client/src/views/Login.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+    FormControl,
+    InputLabel,
+    OutlinedInput,
+    InputAdornment,
+    IconButton,
+    Button,
+    createTheme,
+    ThemeProvider,
+} from '@mui/material'
+import { Visibility, VisibilityOff } from '@mui/icons-material'
+import { useNavigate } from 'react-router-dom'
+import { useState, MouseEvent } from 'react'
+
+export const Login = () => {
+    const theme = createTheme({
+        spacing: 14,
+    })
+    const [showPassword, setShowPassword] = useState(false)
+    const handleClickShowPassword = () => setShowPassword(show => !show)
+    const handleMouseDownPassword = (event: MouseEvent<HTMLButtonElement>) => {
+        event.preventDefault()
+    }
+    const [password, setPassword] = useState('')
+    const handleOnChangePassword = (event: any) => {
+        setPassword(event.target.value)
+    }
```

Comment:
Another way to do this
have one func  e.g. handleChange (used by both user/pass)
and change value base on the field name. That would be useful for bigger forms
In this case this is more reliable, so

Reply:
Ah, mb. I quickly see it when you point it out. Should I try and implement it?

Also, I'm working in the conflict right now. I'll ask if I need help

Reply:
No worries, just wanted to point it out. Since it's only 2 fields not a big deal, either way is fine